cornellGrading/cornellInterface.py

```
@@ -2,11 +2,12 @@
 
 from sys import platform
 
-if platform == "linux" or platform == "linux2" or platform == "darwin":
+if platform.startswith('linux') or platform == "darwin":
     from bullet import Bullet
 elif platform == "win32":
-    from consolemenu import *
-    from consolemenu.items import *
+    from consolemenu import SelectionMenu
+else: 
+    raise Exception('Unsupported Platform')
```

please make this `raise EnvironmentError('Unsupported Platform')`